src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import Tooltip from 'rc-tooltip';
+import DefaultTooltip from 'shared/components/DefaultTooltip';
```

Comment:
small thing but I've stopped using the shared alias b/c the IDE doesn't seem to alble to resolve it.  let me know if it does for you.  i just do relative path to shared.  this gives me code completion on the exports of these modules